explore-reports-and-create-documentation/audit-reports/audit-dashboard.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Audit Dashboard
+description: This article describes Audit Dashboard and its features.
+author: Janko Ban
+date: 14/12/2018
+---
+Audit Dashboard is giving you an overview of the audit actions in the farm for the last 7 days, drillable to specific detailed reports.
+
+Audit Dashboard makes monitoring of the audit actions easier and more intuitive.
+
+The following audit actions are visible:
```

Comment:
ovdje možda umjesto audit actions te dijelove na dashboardu nazvati 'tiles'? - pogledati kako se to zove kod insightsa, ssm-a